app/grants/models/grant.py

```diff
@@ -488,7 +488,7 @@ class Meta:
     )
 
     in_active_clrs = models.ManyToManyField(
-        GrantCLR,
+        "GrantCLR",
```

Comment:
Should this be in ""

Reply:
yea... i guess it's for lazy loading to prevent circular import

Reply:
Yep, that's right @chibie